lib/sqli/sqli_parser.y

```diff
@@ -207,7 +233,16 @@ op_expr:  expr_common
             sqli_store_data(ctx, &$tk);
             YYUSE($u1);
         }
-        | op_expr post_exprs
+        | op_expr in_boolean_mode_exprs
```

Comment:
It is unclear why did you change that.

Reply:
TOK_END (from `post_exprs`) is now used in if-else `expr`